docs/content/contributing/community/office-hours.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Office Hours"
+date: 2018-09-28T10:40:50-07:00
+weight: 3
+---
+
+Athens developers get together for an hour or so approximately every week on a video chat to learn and discuss a part of the codebase. The hour is relatively unstructured, but generally it starts out with a core maintainer (usually [@arschles](https://github.com/arschles)) going into the details of a part of the codebase. From there, we just go where the questions take us.
+
+This is a great opportunity for new community members to get more involved with the project in a low-pressure setting. It's also a great setting for existing community members to gain a deeper understanding into the codebase.
+
+We meet at this [zoom](https://zoom.us/) video Chat URL: [https://arschles.com/zoom](https://arschles.com/zoom)
+
+>**Absolutely everybody is welcome to attend these meetings. You're free to suggest topics to talk about, share your perspective, and participate as little or much as you'd like**.
+>
+>You don't have to be a contributor to attend.
+
+# Logistics
+
+All office hours happen using the [zoom](https://zoom.us) software. Before you come to the meeting, please make sure you have installed the zoom client using this link: https://zoom.us/support/download.
```

Comment:
We should probably wrap the `/download` link in markdown

Reply:
@jpreese after a long delay, I updated this 😆 . Can you let me know what you think?

Reply:
No worries! LGTM :)